networking/src/ModelHelper.ts

```diff
@@ -120,6 +120,7 @@ export abstract class ModelHelper {
         ))
     }
     static calculateBasicModeScoreForQuestion(gameSession: IGameSession, question: IQuestion, team: ITeam, isShortAnswerEnabled: boolean) {
+        console.log("MADE IT IN");
```

Comment:
delete

Reply:
console.logs taken care of. except the help and x icon on host_v2 lobby since we dont have functionality for them yet